urllib3/response.py

```diff
@@ -182,6 +192,38 @@ def tell(self):
         """
         return self._fp_bytes_read
 
+    def _init_length(self, request_method):
+        """
+        Set initial length value for Response content.
+
+        Logic extracted from httplib.py in the standard Python library.
+        """
+        length = self.headers.get('content-length')
+        if length is not None and not self.chunked:
+            try:
+                length = int(length)
```

Comment:
By rejecting, do you mean raising an exception in that event instead of setting `length` to None?

If we want to handle the case (which I think would be better than the exception), something like:

```python
single_length = length.split(',')[0].strip()
length = int(single_length)
```
should work, right?